src/map/lua/luautils.cpp

```diff
@@ -562,11 +562,30 @@ namespace luautils
             return -1;
         }
 
-        lua_getglobal(LuaHandle, "SetRegionalConquestOverseers");
+        lua_getglobal(LuaHandle, "dsp");
         if (lua_isnil(LuaHandle, -1))
         {
             lua_pop(LuaHandle, 1);
-            ShowError("luautils::SetRegionalConquestOverseers: undefined procedure SetRegionalConquestOverseers\n");
+            ShowError("luautils::SetRegionalConquestOverseers: undefined global dsp\n");
+            return -1;
+        }
+
+        int TableIndex = 0;
+        TableIndex = lua_gettop(LuaHandle);
+        lua_getfield(LuaHandle,TableIndex,"conquest");
```

Comment:
should be able to just do -1 for TableIndex, since you just pushed it to the top (-1 = top)